debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -462,6 +462,10 @@ protected void handleQueryEvent(Event event) throws InterruptedException {
             // This is an XA transaction, and we currently ignore these and do nothing ...
             return;
         }
+        if (context.ddlFilter().test(sql)) {
+            logger.info("a DDL '{}' was filtered out of processing", sql);
```

Comment:
I'd remove the "a". And should it be rather `debug` then `info`? For info we say " for status and general activity (usually low-volume)", so that wouldn't quite match.